scripts/esmfold_inference.py

```diff
@@ -83,6 +86,9 @@ def create_batched_sequence_datasest(
     parser.add_argument(
         "-o", "--pdb", help="Path to output PDB directory", type=Path, required=True
     )
+    parser.add_argument(
+        "-m", "--model_pth", help="Parent path to Pretrained ESM data directory. ", type=Path, default=None
```

Comment:
```suggestion
        "-m", "--model_dir", help="Parent path to Pretrained ESM data directory. ", type=Path, default=None
```

`model_pth` lost a vowel but also suggests direct filename rather than parent dir

Reply:
```suggestion
        "-m", "--model-dir", help="Parent path to Pretrained ESM data directory. ", type=Path, default=None
```
Indeed. 
 Looks like this naming pattern `--my-args` of options is preffered in this repo.  `--model_dir` or `--model_pth` is somehow weird.

Reply:
```suggestion
        "-m", "--model-dir", help="Parent path to Pretrained ESM data directory. ", type=Path, default=None
```
Indeed. 
 Looks like this naming pattern `--my-args` of options is preffered in this repo.  `--model_dir` or `--model_pth` is somehow weird.